CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+* Updated the rotation transpose in contact wrench cone to re-use the same object to avoid additional transpose operations.
```

Comment:
Please indicate the PR as in the examples above. Also, notice that there is a conflict with other changes in the changelog.